Workspace/src/xbox_controller/launch/state_select.launch

```diff
@@ -0,0 +1,21 @@
+<launch>
+	<node name="teleop_node" pkg="xbox_controller" type="state_select" />
+	
+	<node name="drive_lis_pub" pkg="xbox_controller" type="joy_to_twist" />
+	
+	<node name="joy_launcher" pkg="joy" type="joy_node">
+		<param name="dev" type="string" value="/dev/input/by-id/usb-©Microsoft_Corporation_Controller_15D5CD4-joystick" />
```

Comment:
@lijerryli this looks ugly af. should be a udev rule right?

Reply:
no, the xbox controller didn't work with udev rules due to the wireless dongle supporting 4 controllers. This was the only way to select it.

Reply:
right this is the wireless controller. we should default to the wired controller (js0) and only use the wireless if required

Reply:
so you want me to keep the id argument or switch it back to js0?